tests/features/common/language-selector.feature

```diff
@@ -1,11 +1,17 @@
 Feature: Select language
 
-  Scenario: Change language
+Scenario: Change language
 
-    When I visit `/de/search?query=`
```

Comment:
Should we comment this out instead, at least when put the language selector back in, we will have the same tests as before?

Reply:
I did comment the feature test out (see below). Is that not what you mean?

Reply:
My bad. sorry